pipes.qmd

```diff
@@ -32,12 +32,12 @@ Avoid using the pipe when:
 # Good
 iris |>
   summarize(across(where(is.numeric), mean), .by = Species) |>
-  pivot_longer(-Species, names_to = "measure", values_to = "value") |>
+  pivot_longer(!Species, names_to = "measure", values_to = "value") |>
```

Comment:
Does `-` no longer work? Is it discouraged?

Reply:
it works, but I feel `!` would fit better since e.g. `?dplyr::select` advertises `!` for taking the complement of a set of variables